### PR TITLE
RDoc-1087 Document session.Advanced.DocumentQuery.Boost

### PR DESCRIPTION
**Pull request by ml054**

### CODE REVIEW

File: Documentation/4.0/Raven.Documentation.Pages/indexes/querying/boosting.dotnet.markdown [hunk 1]
```diff
@@ -0,0 +1,48 @@
+﻿# Boosting
+
+A that RavenDB leverages from Lucene is called Boosting. This feature gives you the ability to manually tune the relevance level of matching documents when performing a query.
```
Comment (arekpalinski) [inline Documentation/4.0/Raven.Documentation.Pages/indexes/querying/boosting.dotnet.markdown]: "A that"?

File: Documentation/4.0/Raven.Documentation.Pages/indexes/querying/boosting.dotnet.markdown [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+
+## Boosting in query
+
+Indexing in RavenDB is built upon the Lucene engine that provides a boosting term mechanism. This feature introduces the relevance level of matching documents based on the terms found.
```
Comment (arekpalinski) [inline Documentation/4.0/Raven.Documentation.Pages/indexes/querying/boosting.dotnet.markdown]: The same sentence as above.

File: Documentation/4.0/Raven.Documentation.Pages/indexes/querying/boosting.dotnet.markdown [hunk 3]
```diff
@@ -0,0 +1,48 @@
+﻿# Boosting
+
+A that RavenDB leverages from Lucene is called Boosting. This feature gives you the ability to manually tune the relevance level of matching documents when performing a query.
+
+
+## Boosting in query
+
+Indexing in RavenDB is built upon the Lucene engine that provides a boosting term mechanism. This feature introduces the relevance level of matching documents based on the terms found. 
+Each search term can be associated with a boost factor that influences the final search results. The higher the boost factor, the more relevant the term will be. 
+RavenDB also supports that, in order to improve your searching mechanism and provide the users with much more accurate results you can specify the boost argument.
```
Comment (arekpalinski) [inline Documentation/4.0/Raven.Documentation.Pages/indexes/querying/boosting.dotnet.markdown]: "RavenDB also supports that" - also?

Comment (arekpalinski) [reply Documentation/4.0/Raven.Documentation.Pages/indexes/querying/boosting.dotnet.markdown]: Maybe rephrase to: This way you can improve your searching mechanism and provide users with much more accurate results.

File: Documentation/4.0/Raven.Documentation.Pages/indexes/querying/boosting.dotnet.markdown [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+A that RavenDB leverages from Lucene is called Boosting. This feature gives you the ability to manually tune the relevance level of matching documents when performing a query.
+
+
+## Boosting in query
+
+Indexing in RavenDB is built upon the Lucene engine that provides a boosting term mechanism. This feature introduces the relevance level of matching documents based on the terms found. 
+Each search term can be associated with a boost factor that influences the final search results. The higher the boost factor, the more relevant the term will be. 
+RavenDB also supports that, in order to improve your searching mechanism and provide the users with much more accurate results you can specify the boost argument. 
+
+For example:
```
Comment (arekpalinski) [inline Documentation/4.0/Raven.Documentation.Pages/indexes/querying/boosting.dotnet.markdown]: ## Examples

File: Documentation/4.0/Raven.Documentation.Pages/indexes/querying/boosting.dotnet.markdown [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+{CODE-TABS}
+{CODE-TAB:csharp:DocumentQuery boosting_2_1@Indexes\Querying\Boosting.cs /}
+{CODE-TAB-BLOCK:csharp:RQL}
+from Users
+where boost(startsWith(Name, 'G'), 10) or boost(startsWith(Name, 'A'), 5)
+{CODE-TAB-BLOCK/}
+{CODE-TABS/}
+
+This shows users which name starts with letter 'G' or 'A'. Results which starts with 'G' goes first.
```
Comment (arekpalinski) [inline Documentation/4.0/Raven.Documentation.Pages/indexes/querying/boosting.dotnet.markdown]: goes -> go

File: Documentation/4.0/Raven.Documentation.Pages/indexes/querying/boosting.dotnet.markdown (hunk 1 above)
Comment (ml054) [reply Documentation/4.0/Raven.Documentation.Pages/indexes/querying/boosting.dotnet.markdown]: will remove this paragraph

File: Documentation/4.0/Raven.Documentation.Pages/indexes/querying/boosting.dotnet.markdown (hunk 2 above)
Comment (ml054) [reply Documentation/4.0/Raven.Documentation.Pages/indexes/querying/boosting.dotnet.markdown]: I leave it

File: Documentation/4.0/Raven.Documentation.Pages/indexes/querying/boosting.dotnet.markdown (hunk 3 above)
Comment (ml054) [reply Documentation/4.0/Raven.Documentation.Pages/indexes/querying/boosting.dotnet.markdown]: done

File: Documentation/4.0/Raven.Documentation.Pages/indexes/querying/boosting.dotnet.markdown (hunk 5 above)
Comment (ml054) [reply Documentation/4.0/Raven.Documentation.Pages/indexes/querying/boosting.dotnet.markdown]: done

File: Documentation/4.0/Raven.Documentation.Pages/indexes/querying/boosting.dotnet.markdown (hunk 4 above)
Comment (ml054) [reply Documentation/4.0/Raven.Documentation.Pages/indexes/querying/boosting.dotnet.markdown]: done

**Comment by ml054** [comment]
@arekpalinski Please retest
